src/Utility/CrewSpecs.cs

```diff
@@ -63,6 +63,34 @@ public bool Check(ProtoCrewMember c)
 			return trait.Length == 0 || (c.trait == trait && c.experienceLevel >= level);
 		}
 
+		// return sum of al bonuses for crew on vessel
```

Comment:
How about using ///<summary> instead of a //comment 😉

Reply:
Nice, didn't know that one. One has to apprechiate how MS manages to reinvent wheels all the time. Many other languages use /** ... */ for that...

Reply:
Uses Intelisense, handy because you can hover over the class, method etc elsewhere in the code and see the comment, I now use it on nearly everything 😃

Reply:
Yeah, but this is the poor mans version of what other IDEs offer, and you have to write even more control gibberish to make it work.

Reply:
different languages comment in different ways, better than nothing though 🤷‍♂️